.github/workflows/check-libraries.yaml

```diff
@@ -0,0 +1,13 @@
+name: Check libraries
+
+on:
+  workflow_call:
```

Comment:
There's a `CHARMHUB_TOKEN` secret missing:
```suggestion
  workflow_call:
    secrets:
      CHARMHUB_TOKEN:
        required: true
```

Reply:
It actually is the same one used in the rest of the workflow files, I fixed the name :)

Reply:
Yeah, but you still need to define it here, because it's not a default GH secret.